app/locales/en/translations.js

```diff
@@ -36,6 +36,38 @@ export default {
         pre_moderation: `pre-moderation`,
         post_moderation: `post-moderation`,
     },
+    documentType: {
```

Comment:
These seem like they should live in `ember-osf`'s translations, replacing `preprintWords`